archivebox/main.py

```diff
@@ -422,7 +424,10 @@
     if existing_index:
         print('{green}[√] Done. Verified and updated the existing ArchiveBox collection.{reset}'.format(**ANSI))
     else:
-        # TODO: allow creating new supersuer via env vars on first init
+        if ARCHIVEBOX_USERNAME and ARCHIVEBOX_PASSWORD:
+            print('{green}[+] ARCHIVEBOX_USERNAME and  ARCHIVEBOX_PASSWORD configuration options found. Creating new admin user with username {} and password {}.{reset}'.format(ARCHIVEBOX_USERNAME, ARCHIVEBOX_PASSWORD, **ANSI))
```

Comment:
Ah yes, also wasn't sure if I should log the actual values or not. I figured it might be slightly more convenient for the user and not much more insecure than storing the same values as environment variables.

Reply:
Whoops, that was sloppy! Lesson learned, no pushing changes at 3am